src/promptflow/tests/test_configs/flows/conditional_flow_with_aggregate_bypassed/collect_node.py

```diff
@@ -0,0 +1,8 @@
+from promptflow import tool
+
+# The inputs section will change based on the arguments of the tool function, after you save the code
+# Adding type to arguments and return value will help the system show the types properly
+# Please update the function name/signature per need
+@tool
```

Comment:
As an aggregation node, the input should be `List[Optional[str]]` instead of str, without default value.